cdk/lib/bigquery-acquisitions-publisher.ts

```diff
@@ -97,16 +103,43 @@ export class BigqueryAcquisitionsPublisher extends GuStack {
           }
         : undefined;
 
+    // SQS to Lambda event source mapping
+    const eventSource = new SqsEventSource(queue, {
+      reportBatchItemFailures: true,
```

Comment:
This needs to be set or the queue doesn't receive the failure response from the lambda and failed messaes are not sent to the dead letter queue